Backpack/src/main/java/net/skyscanner/backpack/util/Consumer.kt

```diff
@@ -0,0 +1,3 @@
+package net.skyscanner.backpack.util
```

Comment:
Can we not use the java `Consumer` interface?

Reply:
Nope, it's only available from API 24. Also, I don't want to use interface here and just declare any function with this literal to be Consumer -- this will allow to use functional literals or even use primitive non-autoboxed types.